core/arch/emit_utils_shared.c

```diff
@@ -1328,6 +1328,11 @@ update_indirect_exit_stub(dcontext_t *dcontext, fragment_t *f, linkstub_t *l)
 int
 fragment_prefix_size(uint flags)
 {
+#ifdef AARCH64
+    /* For AArch64, there is no need to save the flags
+     * so we always have the same ibt prefix. */
+    return fragment_ibt_prefix_size(flags);
```

Comment:
I don't think this code needs to be here: use_ibt_prefix should be true always for AARCH64, right?  So this routine was already returning fragment_ibt_prefix_size(flags).  Cleaner to not have an ifdef.